test/run-jit-perform.py

```diff
@@ -0,0 +1,150 @@
+                      help='directory to search for all executables.')
+  parser.add_argument('--no-error-cmdline',
+                      help='don\'t display the subprocess\'s commandline when'
+                      + ' an error occurs', dest='error_cmdline',
+                      action='store_false')
+  parser.add_argument('-p', '--print-cmd',
+                      help='print the commands that are run.',
+                      action='store_true')
+  parser.add_argument('--run-all-exports', action='store_true')
+  parser.add_argument('--host-print', action='store_true')
+  parser.add_argument('--spec', action='store_true')
+  parser.add_argument('-t', '--trace', action='store_true')
+  parser.add_argument('file', help='test file.')
+  parser.add_argument('--enable-saturating-float-to-int', action='store_true')
+  parser.add_argument('--enable-threads', action='store_true')
+  parser.add_argument('--disable-jit', action='store_true')
+  parser.add_argument('--trap-on-failed-comp', action='store_true')
+  options = parser.parse_args(args)
+
+  wast_tool = None
+  interp_tool = None
+  interp_jit_tool = None
+  wast_tool = utils.Executable(
+      find_exe.GetWat2WasmExecutable(options.bindir),
+      error_cmdline=options.error_cmdline)
+  interp_tool = utils.Executable(
+      find_exe.GetWasmInterpExecutable(options.bindir),
+      error_cmdline=options.error_cmdline)
+  interp_tool.AppendOptionalArgs({
+      '--host-print': options.host_print,
+      '--run-all-exports': options.run_all_exports,
+  })
+
+  interp_jit_tool = utils.Executable(
+      find_exe.GetWasmInterpExecutable(options.bindir),
+      error_cmdline=options.error_cmdline)
+  interp_jit_tool.AppendOptionalArgs({
+      '--host-print': options.host_print,
+      '--run-all-exports': options.run_all_exports,
+  })
+
+  wast_tool.AppendOptionalArgs({
+      '-v': options.verbose,
+      '--enable-saturating-float-to-int':
+          options.enable_saturating_float_to_int,
+      '--enable-threads': options.enable_threads,
+  })
+
+  interp_tool.AppendOptionalArgs({
+      '-v': options.verbose,
+      '--run-all-exports': options.run_all_exports,
+      '--trace': options.trace,
+      '--enable-saturating-float-to-int':
+          options.enable_saturating_float_to_int,
+      '--enable-threads': options.enable_threads,
+      '--disable-jit': True,
+  })
+  interp_jit_tool.AppendOptionalArgs({
+      '-v': options.verbose,
+      '--run-all-exports': options.run_all_exports,
+      '--trace': options.trace,
+      '--enable-saturating-float-to-int':
+          options.enable_saturating_float_to_int,
+      '--enable-threads': options.enable_threads,
+      '--trap-on-failed-comp': options.trap_on_failed_comp,
+  })
+
+  wast_tool.verbose = options.print_cmd
+  interp_tool.verbose = options.print_cmd
+  interp_jit_tool.verbose = options.print_cmd
+
+  with utils.TempDirectory(options.out_dir, 'run-interp-') as out_dir:
```

Comment:
The name of the temporary directory should be changed to reflect the fact that we're running a JIT performance test, not an interpreter test.